plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/homepage.spec.js

```diff
@@ -314,22 +314,23 @@ test.describe( 'Homepage tracking banner', () => {
 		).toBeVisible();
 	} );
 
-	test( 'Should not show the "Want more patterns?" banner when tracking is allowed', async ( {
-		baseURL,
-		pageObject,
-	} ) => {
-		await setOption(
-			request,
-			baseURL,
-			'woocommerce_allow_tracking',
-			'yes'
-		);
+	test(
+		'Should not show the "Want more patterns?" banner when tracking is allowed',
+		{ tag: '@skip-on-default-pressable' },
```

Comment:
This banner is always present on Pressable, with the following message:

<img width="354" alt="Screenshot 2024-08-27 at 19 46 34" src="https://github.com/user-attachments/assets/a1dc0621-6b59-43e7-b22d-ea68d107a84a">